### ISSUE TITLE
Error: unknown flag: --from-cronjob

### ISSUE THREAD

**Issue by sruon** [kubernetes/kubernetes#65008]
/kind bug

**What happened**:
kubectl create job --from-cronjob : unknown flag: --from-cronjob

**What you expected to happen**:
A job to be created with the syntax provided by the CLI help.

**How to reproduce it (as minimally and precisely as possible)**:
kubectl create job test --from-cronjob=existingcron

**Anything else we need to know?**:
According to #60039, the proper syntax is --from-cronjob. 
However, it is clear after testing that the right syntax is --from=cronjob/cron (no issue with actual feature)

The CLI help documentation is equally confused and needs to be updated. See the inaccurate USAGE below - while the right syntax is provided twice before.

```
Examples:
  # Create a job from a CronJob named "a-cronjob"
  kubectl create job --from=cronjob/a-cronjob

Options:
      [...]
      --from='': The name of the resource to create a Job from (only cronjob is supported).
      [...]
Usage:
  kubectl create job NAME [--from-cronjob=CRONJOB] [flags] [options]

unknown flag: --from-cronjob
```

Issue #61787 was opened for the same issue and the patch did not fix this.

**Environment**:
- Kubernetes version (use `kubectl version`): 1.10.4
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by CaoShuFeng** [comment]
> Kubernetes version (use kubectl version): 1.10.4

This is fixed in 1.11 by https://github.com/kubernetes/kubernetes/pull/61219

**Comment by jhorwit2** [comment]
@CaoShuFeng it was partially fixed as @sruon points out in that PR; however, it was fully fixed [here](https://github.com/kubernetes/kubernetes/commit/370bdba4cfe7ce4f9b8acde3163b1d8a613a41a2) so i'm closing this issue. 

/close
